spec/controllers/assignments_controller_spec.rb

```diff
@@ -25,11 +25,29 @@
   end
 
   describe 'POST #create', :vcr do
+    before do
+      request.env['HTTP_REFERER'] = 'http://test.host/classrooms/new'
```

Comment:
So we use `redirect_to :back` in the controllers on errors, and the test doesn't know where to go back to. Setting the `request.env['HTTP_REFERER']` solves this problem.